andela/travel_tool_front
# #61374570 Dashboard layout

#### What does this PR do?
Create a basic layout for the dashboard

#### Description of Task to be completed?
    - create a basic dashboard layout
    - add the dashboard to the sidebar
    - configure dashboard routing

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
```bg-image-reupload-edit-161176885```
- Install dependencies:
```yarn install```
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[61374570](https://www.pivotaltracker.com/story/show/61374570)

#### Screenshots (if appropriate)
<img width="1430" alt="screen shot 2018-10-24 at 01 10 20" src="https://user-images.githubusercontent.com/22371596/47394497-c85e7800-d72b-11e8-9644-5aa6432a0646.png">
